Doc/library/io.rst

```diff
@@ -503,6 +515,40 @@ I/O Base Classes
       If the object is in non-blocking mode and no bytes
       are available, ``None`` is returned.
 
+   .. method:: backread(size=-1, /)
+
+      Read up to *size* bytes from the *end* of the object and return them in
```

Comment:
Could you add a note about seek? Also, it should read from the current position.

Reply:
Sure! (it's in my todo list)